core/inject.js

```diff
@@ -97,7 +95,7 @@ Blockly.inject = function(container, opt_options) {
  * @return {!Element} Newly created SVG image.
  * @private
  */
-Blockly.createDom_ = function(container, options) {
+const createDom_ = function(container, options) {
```

Comment:
Even if we were going to export this with a trailing `_`, the internal name should not have one.  (here and several other cases, below).